### ISSUE TITLE
SOM bounding boxes crash on certain websites

### ISSUE THREAD

**Issue by mlin12321** [OSU-NLP-Group/SeeAct#46]
The text_choice_som script will cause SeeAct to crash on certain websites, e.g. https://www.behance.net/, with an unmark_page not defined error. Attached is a screenshot of the error thrown.
![unmarked_page](https://github.com/user-attachments/assets/c4907542-b44f-4d16-8d59-094c8143b534)

**Comment by duz-sg** [comment]
Yes, it was observed in the past, and we did some adjustments to fix this issue as much as possible.
The root cause is that for certain website, it redirects to a new page after few seconds, therefore, when a new page is loaded, all the js that previously loaded are gone, leading to this js function not defined error.
If you see this issue again, please provide the website and your instruction, so we could reproduce this issue.
